staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2143,6 +2143,19 @@ type TCPSocketAction struct {
 	Host string `json:"host,omitempty" protobuf:"bytes,2,opt,name=host"`
 }
 
+type GRPCAction struct {
+	// Port number of the gRPC service. Number must be in the range 1 to 65535.
+	Port int32 `json:"port" protobuf:"bytes,1,opt,name=port"`
+
+	// Service is the name of the service to place in the gRPC HealthCheckRequest
+	// (see https://github.com/grpc/grpc/blob/master/doc/health-checking.md).
+	//
+	// If this is not specified, the default service name is the empty string (i.e. "").
```

Comment:
I swear we changed  the wording of this in the original PR, but I can't find it

for followup, something like

"If this is not specified, the default behavior is defined by gRPC."